vsts_cd_manager/continuous_delivery_manager.py

```diff
@@ -161,6 +161,8 @@ def _get_build_configuration(self, app_type, working_directory):
             build_configuration = BuildConfiguration(app_type, working_directory)
         elif app_type == 'AspNetCore':
             build_configuration = BuildConfiguration(app_type, working_directory)
+        elif app_type == 'PHP':
```

Comment:
This change is just adding a new enum value for 'app_type' parameter. Writing L0 will not add any value here, given that there is test case in-place where other enums are being used.
Test Case Details: -
name -> test_config_source_control_vsts
file -> test_webapp_commands_thru_mock.py